LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Zapier
+Copyright (c) 2018 Zapier Inc.
```

Comment:
Should this be 2019 now :trollface:

Reply:
Yep, that's how long it took to finish this PR. 😅